IPython/core/debugger.py

```diff
@@ -532,15 +551,62 @@ def print_list_lines(self, filename, first, last):
         except KeyboardInterrupt:
             pass
 
+    def do_skip_predicates(self, args):
+        """
+        Turn on/off individual predicates as to whether a frame should be hidden/skip.
+
+        The global option to skip (or not) hidden frames is set with skip_hidden
+
+        To change the value of a predicate
+
+            skip_predicates key [true|false]
+
+        Call without arguments to see the current values.
+
+        """
+        if not args.strip():
+            print("current predicates:")
+            for (p, v) in self._predicates.items():
+                print("   ", p, ":", v)
+            return
+        type_value = args.strip().split(" ")
+        if len(type_value) != 2:
+            print("Usage: skip_predicates <type> <value>")
```

Comment:
Nit: maybe this should mention possible `<type>`s, just to make it simpler in case someone makes a typo?

Reply:
I've added the possible types, but if the user does a typo it will say:

```
ipydb> skip_predicates a bc
'a' not in {'readonly', 'ipython_internal', 'tbhide'}
```

So this is more if they use an incorrect syntax by putting only one value.